packages/create-jsx-email/starter/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "jsx-email-starter",
```

Comment:
if a user chooses a custom name, this won't be correct.

let's use `mustache` here and rename the file to `package.mustache`